### PR TITLE
V1 finishing touches

### PR DESCRIPTION
**Pull request by rachelbock**
Commits on Apr 25, 2016
    rage    Minor updates to login activity image and other resources.  
Commits on Apr 26, 2016
    rage    Added Toolbar to remove elevation. Modified Layout.  
Commits on May 02, 2016
    rage    Final updates to V1.  …
- Added custom font to Login Activity and Home Page for Circuit and Clamber text.
- Updated Climb Data to remove obsolte integer that was removed from the table.
- Reorganized code for easier navigation in android studio.
- Added drawables from google icons for actions.
- Added new main wall images.
- Added additional comments to various fragments.

### CODE REVIEW

File: app/src/main/java/com/rage/clamber/Activities/HomePage.java [hunk 1]
```diff
@@ -34,49 +43,131 @@ protected void onCreate(Bundle savedInstanceState) {
...
+     * @param tab - the tab that was selected.
      */
-    @OnClick(R.id.action_bar_home_button)
-    public void onHomeButtonClicked(Button button) {
-        FragmentTransaction transaction = getSupportFragmentManager().beginTransaction();
-        transaction.replace(R.id.home_page_frame_layout, HomeFragment.newInstance(user));
-        transaction.addToBackStack(null);
-        transaction.commit();
+    public void tabSelected(TabLayout.Tab tab) {
+        if (tab.getPosition() == 0){
```
Comment (nonvirtualthunk) [inline app/src/main/java/com/rage/clamber/Activities/HomePage.java]: Let's unify the shared parts of these four branches

File: app/src/main/java/com/rage/clamber/Activities/HomePage.java [hunk 2]
```diff
@@ -34,49 +43,131 @@ protected void onCreate(Bundle savedInstanceState) {
...
-        transaction.replace(R.id.home_page_frame_layout, ProjectsFragment.newInstance(user));
-        transaction.addToBackStack(null);
-        transaction.commit();
+    /**
+     * Method to clear the backstack. Used to ensure fragments are not building up. Called each time
+     * a tab is changed.
+     */
+    private void clearBackstack() {
+        FragmentManager manager = getSupportFragmentManager();
+        if (manager.getBackStackEntryCount() > 0) {
```
Comment (nonvirtualthunk) [inline app/src/main/java/com/rage/clamber/Activities/HomePage.java:130]: It seems like this just removes the first element from the backstack, is that intention, or does `manager.popBackStack` do something other than just remove the specific backstack entry you give it?

File: app/src/main/java/com/rage/clamber/Fragments/HomeActivity/Walls/ClimbDetailFragment.java [hunk 3]
```diff
@@ -304,20 +316,31 @@ public void onFailure(Call<Double> call, Throwable t) {
...
      */
-    public void addRatingForUser(UserRatingRequest request){
+    public void addRatingForUser(UserRatingRequest request) {
         final Call<Integer> addRatingCall = ApiManager.getClamberService().addUserRating(request);
         addRatingCall.enqueue(new Callback<Integer>() {
             @Override
             public void onResponse(Call<Integer> call, Response<Integer> response) {
                 if (response.code() == 200) {
-                    yourRatingTextView.setText(getContext().getString(R.string.your_rating_s, response.body()));
+                    if (response.body() == -1){
```
Comment (nonvirtualthunk) [inline app/src/main/java/com/rage/clamber/Fragments/HomeActivity/Walls/ClimbDetailFragment.java]: This is essentially the same code as above for the userRating text, go ahead and make a function that converts a raw double into an avg rating string, and a function that converts an int into an appropriate rating string. Death to code duplication!

File: app/src/main/java/com/rage/clamber/Activities/HomePage.java (hunk 1 above)
Comment (rachelbock) [inline app/src/main/java/com/rage/clamber/Activities/HomePage.java]: Ok

File: app/src/main/java/com/rage/clamber/Activities/HomePage.java (hunk 2 above)
Comment (rachelbock) [inline app/src/main/java/com/rage/clamber/Activities/HomePage.java:130]: it removes the whole backstack.

File: app/src/main/java/com/rage/clamber/Fragments/HomeActivity/Walls/ClimbDetailFragment.java (hunk 3 above)
Comment (rachelbock) [inline app/src/main/java/com/rage/clamber/Fragments/HomeActivity/Walls/ClimbDetailFragment.java]: ok

**Comment by nonvirtualthunk** [comment]
Excellent, merging

**Pull request merged**